sonar-plugin-api/src/main/java/org/sonar/api/scan/issue/filter/FilterableIssue.java

```diff
@@ -55,6 +53,10 @@
   @CheckForNull
   Double gap();
 
+  /**
+   * @deprecated since 6.6 useless
+   */
+  @Deprecated
   Date creationDate();
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)